--- Crypt-CBC-2.14/eg/aes.pl.orig	2005-04-22 19:40:33.000000000 +0200
+++ Crypt-CBC-2.14/eg/aes.pl	2005-06-01 21:02:53.117951992 +0200
@@ -1,7 +1,5 @@
 #!/usr/bin/perl
 
-use lib '../blib/lib';
-
 use Getopt::Std;
 use Crypt::CBC;
 use strict vars;
--- Crypt-CBC-2.14/eg/des.pl.orig	2005-04-22 19:39:45.000000000 +0200
+++ Crypt-CBC-2.14/eg/des.pl	2005-06-01 21:02:55.919526088 +0200
@@ -1,7 +1,5 @@
 #!/usr/bin/perl
 
-use lib '../blib/lib';
-
 use Getopt::Std;
 use Crypt::CBC;
 use strict vars;
--- Crypt-CBC-2.14/eg/idea.pl.orig	2005-04-22 19:39:50.000000000 +0200
+++ Crypt-CBC-2.14/eg/idea.pl	2005-06-01 21:02:58.462139552 +0200
@@ -1,7 +1,5 @@
 #!/usr/bin/perl
 
-use lib '../blib/lib';
-
 use Getopt::Std;
 use Crypt::IDEA;
 use Crypt::CBC;
